libbeat/dashboards/import_dashboards.go

```diff
@@ -211,7 +211,7 @@ func (imp Importer) CreateIndex() error {
 			},
 		})
 	if err != nil {
-		fmt.Printf("fail to set the mapping. Error: %s\n", err)
+		fmt.Printf("failed to set the mapping. Error: %s\n", err)
```

Comment:
Ignoring the error here is by intention as `CreateIndex` might return an error if the index already has those settings. It might make sense to prepend "Info:" or "Warning:" to it to avoid confusion.